### ISSUE TITLE
Include sync of Manager from AD

### ISSUE THREAD

**Issue by pzljanb** [Puzzlepart/did#1244]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
There is a need for mangers to get overview of the status of subordinates when it comes to:
- Confirmed weeks
- Reporting in general

### Description
Sync manager information with AD, and add information to user profile

With this enhancement managers can keep track of the subordinates timesheets, both for follow up and progress reporting

With this improvement managers and users get closer, and manager can keep track of his/hers subordinates progress when it comes to delivery of timesheets each period. 

Describe the functionality in as much detail as you can, feel free 
to add sketches and screenshots.

- [x] Include manger to be synced with AD, and related with user in did
- [x] Include information of manager in the user profile
- [ ] Include information of subordinates in a "Manager" view. Aa a new tab "My team".
- [x] Add Manager as column in reports where users are listed to enable managers to filter and group on this column

**Comment by olemp** [comment]
Missing _Include information of subordinates in a "Manager" view. Aa a new tab "My team"._ @pzljanb. I propose to create a separate issue for this with more details.
